web/src/pages/Profile/Votes/VoteCard/index.tsx

```
@@ -0,0 +1,80 @@
+const ReStyledArrowLink = styled(StyledArrowLink)`
+  font-size: 14px;
+
+  > svg {
+    height: 15px;
+    width: 15px;
+  }
+`;
+
+interface IVoteCard {}
+
+const VoteCard: React.FC<IVoteCard> = ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)